tests/integration/tier0/non-destructive/subscription-manager/main.fmf

```diff
@@ -18,13 +18,8 @@ tag+:
         removed during the conversion run. This test makes sure that subscription-manager and its dependencies are correctly
         backed up and re-installed during the rollback together with the certificate.
 
-    adjust+:
-        - enabled: true
-          when: >
-              distro == centos-8
-
     environment+:
-        TEST_REQUIRES: subscription-manager
+        C2R_TESTS_SUBMAN_CLEANUP: 1
```

Comment:
This is kind of no-op.
The only place this envar is used at is the `pre_registered` fixture, which honestly does not really make sense anymore, given we have the `C2R_TESTS_NONDESTRUCTIVE` envar exported for all non-destructive tests.
So I would remove the whole `environment+:` from here, change the condition in the `pre_registered` to utilize `C2R_TESTS_NONDESTRUCTIVE` and remove `C2R_TESTS_SUBMAN_CLEANUP` everywhere, please.

Reply:
Sounds nice, yeah we should remove this `C2R_TESTS_SUBMAN_CLEANUP` env var and replace it with the more generic `C2R_TESTS_NONDESTRUCTIVE`